.github/ISSUE_TEMPLATE/TASK.yaml

```
@@ -1,11 +1,75 @@
----
-name: 🏃🏻‍♂️ Task
-description: A general task that does not fit into other categories.
-title: "[task]: "
-type: Task
+name: "Task"
+description: "Work item that goes through refinement before development"
+title: "[Task] <short title>"
+labels: ["task"]
+
 body:
   - type: textarea
-    id: description
+    id: purpose
     attributes:
-      label: Description
-      description: Give a brief description of the task.
+      label: Purpose
+      description: Briefly explain why this task is important and what it delivers.
+      placeholder: |
+        Briefly describe what the task should deliver and why it matters.
+        e.g. Add caching to CI pipelines to reduce build time.
+    validations:
+      required: true
+
+  # --- Do we want to estimate the size of the task? If yes, this is my suggestion,
+  # since we used T-shirt sizes in the old DevOps team and it worked well.
+  # This combines that approach with a bit of planning poker. ---
+  - type: dropdown
+    id: size
+    attributes:
+      label: Size (T-shirt)
+      options: [XS, S, M, L, XL]
+    validations:
+      required: true
+
+  - type: textarea
+    id: acceptance
+    attributes:
+      label: Acceptance Criteria
+      description: What must be true for the task to be considered complete.
+      placeholder: |
+        Examples:
+        - [ ] When the pipeline runs, the new caching is used and reduces build time.
+        - [ ] When the feature flag is enabled, the new UI is visible to all users.
+        - [ ] When Stefan can press the button without getting an error, we are done.
+
+  - type: textarea
+    id: dependencies
+    attributes:
+      label: Dependencies
+      description: Link to other issues, external reviews, etc.
+      placeholder: |
+        - Blocked by: #123
+        - Blocks: #456
+
+  - type: textarea
+    id: design
+    attributes:
+      label: Design / Solution Direction
+      description: Short note or link to diagram/ADR if relevant.
+
+  # --- Not sure if we really need this, but it was part of the given requirements. ---
+  - type: textarea
+    id: data_env
+    attributes:
+      label: Data & Environments
+      description: Note any required data, access, or environment setup.
+      placeholder: "e.g. Test dataset, staging environment configuration…"
```

I think this makes more sense as general prerequisites. 

```suggestion
  - type: textarea
    id: prerequisites
    attributes:
      label: Prerequisites
      description: Note any prerequisites like required data, accesses, or environment setup.
      placeholder: |
        - [ ] Administrative rights to an AWS Account
        - [ ] Data on PRs merged in march.
        - [ ] A tenant on the Developer Cloud Platform.
```